R/utilities.R

```diff
@@ -1631,6 +1689,11 @@ collate_results = function(sample_table,
 
     full_table = left_join(meta_data, sample_table)
 
+    #check for missing columns, add any missing columns and fill with NA
+    col_check = c("ashm_MYC", "manta_MYC_sv", "ICGC_MYC_sv", "myc_ba", "ashm_BCL2", "manta_BCL2_sv", "ICGC_BCL2_sv", "bcl2_ba") #create a vector of the columns to check for
```

Comment:
Yeah, I think that's the way to go. This can be considdered as a temporary fix for the problem. But we should probably investigate the underlying cause for the issue.